microprofile/WFLY-12681_Provide_a_microprofile-platform_Galleon_layer.adoc

```diff
@@ -0,0 +1,86 @@
+= WFLY-12681 Provide a 'microprofile-platform' Galleon layer
+:author:            Radoslav Husar
+:email:             rhusar@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Historically, we used to have a 'microprofile' layer spec aggregating all MicroProfile (further as MP) specs.
+At that moment it only contained config, health and metrics which got eventually renamed
+to 'observability' by https://issues.redhat.com/browse/WFLY-11774[WFLY-11774] (no RFE there).
+
+With the introduction of new the specs such as MP Fault Tolerance, MP OpenAPI, etc. we want to
+reintroduce the 'microprofile-platform' layer spec which contains all available MP specs in the server.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-12681[WFLY-12681 Provide a 'microprofile-platform' Galleon layer]
+* https://issues.redhat.com/browse/EAP7-1396[EAP7-1396 Provide a 'microprofile-platform' Galleon layer]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY-11774[WFLY-11774 Rename microprofile layer to be observability]
+
+=== Dev Contacts
+
+* mailto:rhusar@redhat.com[Radoslav Husar]
+
+=== QE Contacts
+
+* TBD
+
+=== Testing By
+
+* [x] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
```

Comment:
@tommaso-borgato Well, all of these pertain to WildFly project so that seems very unnecessary. But if that helps to differentiate between `wildfly` repo and `wildfly-core` repo, why not. Also, moved the other projects to "interested" as they are not directly affected by this but will be of interest 'for them'.